libs/components/forms/src/lib/modules/input-box/input-box.component.ts

```diff
@@ -220,6 +220,7 @@ export class SkyInputBoxComponent
     this.#inputBoxHostSvc.init(this);
   }
 
+  // eslint-disable-next-line complexity
```

Comment:
Should we be disabling this - I know it is annoying in the editor - but I thought we added this as a warning intentionally - and we should really be fixing these eventually instead of disabling them.

Maybe a flex topic? But feels like we shouldn't disable this here.